zest/releaser/release.py

```diff
@@ -121,10 +121,14 @@ def _upload_distributions(self, package):
             result = utils.execute_command(utils.setup_py('bdist_wheel'))
         utils.show_interesting_lines(result)
         if not self.pypiconfig.is_pypi_configured():
-            logger.warn("You must have a properly configured %s file in "
-                        "your home dir to upload to a package index.",
-                        pypi.DIST_CONFIG_FILE)
-            return
+            logger.error(
+                "You must have a properly configured %s file in "
+                "your home dir to upload to a package index.",
```

Comment:
This is only for Python packages yes, so only if there is a `setup.py`. But I might as well mention Python here, done.